packages/runtime/src/widgets/Search.tsx

```diff
@@ -227,6 +227,7 @@ export const Search: React.FC<SearchProps> = ({
           renderLabel(label, labelValue)
         }
         notFoundContent={notFoundContentRenderer}
+        onBlur={handleClear}
```

Comment:
shouldn't we expose the onBlur property with EnsembleAction instead of clearing the results.

Clearing means user will lost the previously searched results, this might solve the current problem, but not a general use case

CC: @evshi

Reply:
The issue we are trying to solve here is if the value is emptied, then the search results should also be emptied. It doesn't necessarily have anything to do with `blur`. Ticket for context https://atlashealth.atlassian.net/browse/TRINITY-1461

@justEhmadSaeed what is clearing the search input value?

Reply:
In that case, we should add an effect/fix the effect that syncs the search results with the value. If the value was set to be null, the search results should get cleared @justEhmadSaeed

Reply:
@evshi updated the PR